actors/builtin/miner/miner_actor.go

```diff
@@ -2169,18 +2235,21 @@ func (a Actor) ProveReplicaUpdates(rt Runtime, params *ProveReplicaUpdatesParams
 			continue
 		}
 
+		var activateRes market.ActivateDealsReturn
 		code := rt.Send(
 			builtin.StorageMarketActorAddr,
 			builtin.MethodsMarket.ActivateDeals,
 			&market.ActivateDealsParams{
-				DealIDs:      update.Deals,
-				SectorExpiry: sectorInfo.Expiration,
+				Sectors: []market.ActivateDealsParamsInner{{
+					DealIDs:      update.Deals,
+					SectorExpiry: sectorInfo.Expiration,
+				}},
```

Comment:
We should batch this too, and remove the call to `requestDealWeights` below in favour of using this return value. That's a nice saving.